servicepulse/event-types.md

```diff
@@ -0,0 +1,37 @@
+---
+title: ServicePulse events
+summary: 'Introduction to ServicePulse monitoring events'
+tags: [ServicePulse, Concepts, Monitoring Events]
+---
+
+ServicePulse is the Particular platform monitoring front-end, powered by ServiceControl. ServicePulse gives you an overview of the system health, based on endpoints heartbeats and custom checks, and a detailed view of all the failed messages.
+
+Note: You can consume the same information not only via the ServicePulse web interface but also subscribing to ServicePulse events broadcasted by the ServiceControl endpoint.
+
+### Heartbeats
```

Comment:
not sure if this http://docs.particular.net/servicepulse/custom-notification-and-alerting-using-servicecontol-events (which @sfarmar did recently) effects this page??